apps/mobile/src/app/account/[account]/index.tsx

```diff
@@ -19,15 +20,20 @@ export default function AccountScreen() {
   const isErrorTotalBalance = aggregateBalance.state === 'error';
   const totalBalance = aggregateBalance.state === 'success' ? aggregateBalance.value : EmptyBalance;
   return (
-    <AccountLoader fingerprint={fingerprint} accountIndex={accountIndex}>
-      {account => (
-        <Account
-          account={account}
-          balance={totalBalance}
-          isLoading={isLoadingTotalBalance}
-          isError={isErrorTotalBalance}
-        />
+    <WalletLoader fingerprint={fingerprint}>
```

Comment:
Maybe there's a better way than this double loader?

Reply:
I have gotten used to the loader pattern and have quite a few nested in the extension sometimes for the rpc requests. I really haven't found a better solution.

Reply:
OK great, just checking in case there is a better way I don't know yet. I actually like this pattern. 

Thanks a lot! 🙇